### PR TITLE
Add missing organization resource limits

### PR DESCRIPTION
**Pull request by falzm**
This change adds missing resource limits to the `exo limits` output.

### CODE REVIEW

File: cmd/limits.go [hunk 1]
```diff
@@ -28,94 +45,140 @@ var limitsCmd = &cobra.Command{
...
+							instance.InstanceTypeID,
+							err)
+					}
+					instanceTypes[instance.InstanceTypeID] = instanceType
+				}
+
+				if strings.HasSuffix(instanceType.Family, "gpu") {
+					cur, _ = curUsage.Load(limitGPUs)
+					curUsage.Store(limitGPUs, cur.(int)+int(instanceType.GPUs))
+				}
```
Comment (zyegfryed) [inline cmd/limits.go:95]: AFAIU it means that a running GPU instance is counted twice in limits: for instance limits and GPU limits, right?

Comment (zyegfryed) [reply cmd/limits.go:95]: Answering to myself by practicing: Yes

```
❯ gco ch30748
Branch 'ch30748' set up to track remote branch 'ch30748' from 'origin' by rebasing.
Switched to a new branch 'ch30748'
❯ make build
[...]
❯ ./bin/exo config
✔ prd*
❯ ./bin/exo limits
┼────────────────────────┼──────┼─────┼
│        RESOURCE        │ USED │ MAX │
┼────────────────────────┼──────┼─────┼
│ IP addresses           │ 0    │ 5   │
│ IAM API keys           │ 3    │ 100 │
│ GPUs                   │ 0    │ 1   │
│ Private Networks       │ 0    │ 32  │
│ Instances              │ 1    │ 20  │
│ Templates              │ 0    │ 30  │
│ Network Load Balancers │ 0    │ 5   │
│ Snapshots              │ 0    │ 30  │
│ SKS clusters           │ 0    │ 3   │
│ SOS buckets            │ 1    │ 100 │
┼────────────────────────┼──────┼─────┼
❯ ./bin/exo vm create gpu-001 -o GPU-small
Creating single-use SSH key
Deploying "gpu-001" ⠏                                                                                done
[...]
❯ ./bin/exo limits
┼────────────────────────┼──────┼─────┼
│        RESOURCE        │ USED │ MAX │
┼────────────────────────┼──────┼─────┼
│ IP addresses           │ 0    │ 5   │
│ IAM API keys           │ 3    │ 100 │
│ GPUs                   │ 1    │ 1   │  <-- GPU count incremented as expected
│ Private Networks       │ 0    │ 32  │
│ Instances              │ 2    │ 20  │ <-- Instances count incremented too
│ Templates              │ 0    │ 30  │
│ Network Load Balancers │ 0    │ 5   │
│ Snapshots              │ 0    │ 30  │
│ SKS clusters           │ 0    │ 3   │
│ SOS buckets            │ 1    │ 100 │
┼────────────────────────┼──────┼─────┼
```

**Review by zyegfryed — APPROVED** [review]
🚀

File: cmd/limits.go (hunk 1 above)
Comment (falzm) [reply cmd/limits.go:95]: @zyegfryed indeed, `GPUs` refers here to the number of GPU cards utilized in Compute instances within the organization, it is not a *type* of Compute instance.

**Review by retrack — APPROVED** [review]
Tested and works as expected. Thanks @falzm for the quick fix!

**Pull request merged**